packages/react-magma-dom/scripts/generateJSON.js

```diff
@@ -83,9 +83,11 @@ const filterJson = () => {
 
     return sortObject(
       children
+        .filter(a => a)
         .filter(filterReactDefinitions)
         .filter(filterMotionDefinitions)
         .filter(filterEmotion)
+        .filter(filterExtendedDefinitions(['BasePaginationProps']))
```

Comment:
I know we only care about this one for now... but we should move it into a tag for the Interface... we do that for ```@children true``` already